### DISCUSSION TITLE
Need to set the fix resolution

### DISCUSSION THREAD

**Discussion by ChiragLiveu** [flutterjanus/flutter_janus_client#157] [Q&A]
Is there a solution to address the resolution issue? I need to disable adaptive resolution support, and I'm wondering if there's a specific flag to set it to false and remove the configuration. Despite extensive research and code debugging, I haven't been able to find a solution. Can anyone assist with this?

**Comment by shivanshtalwar0** [comment]
I think you are asking in context to google meet example simulcasting, that is on receiver end we see (high|medium|low) buttons and you want to get rid of them?
so before you do that you first need to remove `simulcastSendEncodings` because this property is responsible for enabling simulcasting
```dart
    localVideoRenderer.mediaStream = await videoPlugin?.initializeMediaDevices(simulcastSendEncodings: [
      RTCRtpEncoding(rid: "h", minBitrate: 2000000, maxBitrate: 2000000, active: true, scalabilityMode: 'L1T2'),
      RTCRtpEncoding(
        rid: "m",
        minBitrate: 1000000,
        maxBitrate: 1000000,
        active: true,
        scalabilityMode: 'L1T2',
        scaleResolutionDownBy: 2,
      ),
      RTCRtpEncoding(
        rid: "l",
        minBitrate: 512000,
        maxBitrate: 512000,
        active: true,
        scalabilityMode: 'L1T2',
        scaleResolutionDownBy: 3,
      ),
    ], mediaConstraints: {
      'video': {'width': 1280, 'height': 720},
      'audio': true
    });
    ```
    secondly change the flutter code to remove those simulcast buttons
